src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -27,6 +28,7 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Prefer32Bit>false</Prefer32Bit>
+    <NoWarn>PCR0001</NoWarn>
```

Comment:
It looks like we have a number of tests that aren't using `ConfigureAwait`. How critical do you think enforcing that is vs. just adding this ignore?